### ISSUE TITLE
allow_worker_ssh_access=true has no effect

### ISSUE THREAD

**Issue by mailbox171** [oracle-terraform-modules/terraform-oci-oke#494]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform Version and Provider Version

Hello,
I am using versio 4.1.5

source                                =   "oracle-terraform-modules/oke/oci"
version                               =   "4.1.5"
  
  
Terraform v1.1.5
on linux_amd64
+ provider registry.terraform.io/hashicorp/cloudinit v2.2.0
+ provider registry.terraform.io/hashicorp/local v2.2.2
+ provider registry.terraform.io/hashicorp/null v3.1.0
+ provider registry.terraform.io/hashicorp/oci v4.67.0
+ provider registry.terraform.io/hashicorp/time v0.7.2

### Affected Resource(s)

workers nsg

### Terraform Configuration Files

```hcl
# Copy-paste your Terraform configurations here - for large Terraform configs,
# please use a service like Dropbox and share a link to the ZIP file. 
# Please remove any sensitive information from configuration files before sharing them. 
```

### Expected Behavior

Since I set "allow_worker_ssh_access=true",
The workers nsg should have Ingress for port 22 from bastion, I guess

### Actual Behavior

There is no Ingress for port 22 from CIDR including bastion (my vcn CIDR is 10.100.0.0/16, my bastion IP is 10.100.0.5)
(More: there isn't ANY Ingress for port 22 for workers)

ssh from bastion to workers just hangs

### Steps to Reproduce

1. `terraform apply`

**Comment by hyder** [comment]
@mailbox171: fix incoming in #498

**Comment by hyder** [comment]
fixed in #498
